packages/webpack/src/plugin.js

```diff
@@ -255,6 +257,16 @@ class LavaMoatPlugin {
           )
         )
 
+        // Adjust scuttling configuration to not scuttle webpack chunk loading facilities
+        if (
+          typeof STORE.options.scuttleGlobalThis === 'object' &&
+          STORE.options.scuttleGlobalThis.exceptions
```

Comment:
[suggestion] if you're gonna check types, might as well check types

```suggestion
          Array.isArray(STORE.options.scuttleGlobalThis.exceptions)
```